js/src/components/account-card/index.scss

```diff
@@ -9,16 +9,29 @@
 		}
 	}
 
+	&__title:not(:empty) {
+		margin-bottom: 4px;
+		color: $black;
+	}
+
 	&__description {
 		display: flex;
 		flex-direction: column;
 		gap: 1em;
+		color: $gray-900;
 
 		> p {
 			margin: 0;
 		}
 	}
 
+	&__helper {
+		margin-top: 4px;
```

Comment:
💅 An alternative to this would be to apply `margin-bottom: 4px` to `&__description:not(:last-child)`, if we want to use top-down flow with `margin-bottom`.

Reply:
Thanks for the suggestion! I think we can keep it as is since the `helper` is conditionally rendered.